plugins/woocommerce/templates/order/attribution-details.php

```diff
@@ -82,6 +82,33 @@
 			</span>
 		<?php endif; ?>
 
+		<?php if ( array_key_exists( 'utm_source_platform', $meta ) ) : ?>
+		<h4>
+			<?php esc_html_e( 'Source platform', 'woocommerce' ); ?>
+		</h4>
+		<span class="order-attribution-utm-source-platform">
+				<?php echo esc_html( $meta['utm_source_platform'] ); ?>
+			</span>
+		<?php endif; ?>
```

Comment:
The formatting isn't quite right here?

```suggestion
		<?php if ( array_key_exists( 'utm_source_platform', $meta ) ) : ?>
			<h4>
				<?php esc_html_e( 'Source platform', 'woocommerce' ); ?>
			</h4>
			<span class="order-attribution-utm-source-platform">
				<?php echo esc_html( $meta['utm_source_platform'] ); ?>
			</span>
		<?php endif; ?>
```

Reply:
Note that this has been fixed in [`0f489ff` (#47639)](https://github.com/woocommerce/woocommerce/pull/47639/commits/0f489ffba0a5150952c53cf45306e678c9a5449e).